docs/modules/ROOT/pages/how-tos/change-parameter.adoc

```diff
@@ -0,0 +1,37 @@
+= Change a Parameter
+
+Suppose you want to overwrite an inventory parameter for a cluster or tenant.
+This page assumes that you have already set up access to Lieutenant, so that `commodore catalog compile` generally works from any directory.
+
+. Get the Cluster and Tenant ID from Lieutenant and compile the catalog.
++
+[source,bash]
+----
+export $CLUSTER_ID = <target-cluster-id>
+dir=$(mktemp -d)
+pushd "${dir}"
+commodore catalog compile $CLUSTER_ID
+----
+. Open your editor and edit the files you need to change.
++
+TIP: Most probably the file is in `inventory/classes/<tenant-id>`
```

Comment:
```suggestion
TIP: If you want to overwrite an inventory parameter for a tenant or cluster, edit the appropriate file in `inventory/classes/<tenant-id>`.
If you want to adjust a parameter for a Kubernetes distribution or cloud provider, edit the appropriate file in `inventory/classes/global`.
+
TIP: You can compile the catalog locally to verify your changes with `commodore catalog compile $CLUSTER_ID --local`.
```

Reply:
Any reason why you didn't include the 2nd tip (running `commodore catalog compile $CLUSTER_ID --local`) in the current version?

Reply:
no, I overlooked that :see_no_evil:

Reply:
better?